.github/workflows/digit-ui.yaml

```
@@ -44,3 +43,16 @@ jobs:
           docker build  -t selcohub/micro-ui:${{ env.VERSION }}-${{ env.commit_hash }} -f  web/docker/Dockerfile .
           docker push selcohub/micro-ui:${{ env.VERSION }}-${{ env.commit_hash }}
         working-directory: frontend/micro-ui
+
+Workflow_Trigger:
```

actionlint (1.7.4)

unexpected key "Workflow_Trigger" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)